admin_guide/topics/creating-primed-images-cluster-auto-scaler.adoc

```diff
@@ -0,0 +1,142 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='creating-primed-image-cluster-auto-scaler-{context}']
+= Creating a primed image
+
+You can use Ansible playbooks to automatically create and seal a
+primed image for the auto-scaler to use. You must provide attributes
+from your existing Amazon Web Services (AWS) cluster.
+
+[NOTE]
+====
+If you already have a primed image, you can use it instead of creating a new one.
+====
+
+.Procedure
+
+On the host that you used to create your {product-title} cluster, create a
+primed image:
+
+. Create a new Ansible inventory file on your local host:
++
+----
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[OSEv3:vars]
+ifdef::openshift-enterprise[]
+openshift_deployment_type=openshift-enterprise
+endif::[]
+ifdef::openshift-origin[]
+openshift_deployment_type=origin
+endif::[]
+ansible_ssh_user=ec2-user
+openshift_clusterid=mycluster
+ansible_become=yes
+
+[masters]
+[etcd]
+[nodes]
+----
+
+. Create provisioning file, *_build-ami-provisioning-vars.yaml_*, on your local
+host:
++
+[source,yaml]
+----
+ifdef::openshift-enterprise[]
+openshift_deployment_type=openshift-enterprise
```

Comment:
Should be "openshift_deployment_type: openshift-enterprise"

Reply:
Correct. Not sure what I was thinking yesterday.